src/components/MobileNavigation/MobileNavigation.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import "./MobileNavigation.scss";
+
+function MobileNavigation() {
+    return (
+        <div className='MobileNavigation'>
+            <div className='MobileNavigation__wrapper'>
+                <a href='#' className='MobileNavigation__wrapper__link1'>Find Candidates</a>
+                <a href='#' className='MobileNavigation__wrapper__link2'>Find Jobs</a>
+                <button className='MobileNavigation__wrapper__button'>
+                    <a href='#' target="_blank">Sign In</a>
```

Comment:
do we need a anchor tag here? we can add a onclick listener if needed

Reply:
we can also use just anchor tag, without button:
<a  className="MobileNavigation__button"
          href="#"
          target="_blank">
          Sign In
        </a>
    
and style it like a button